docs/tsc/meetings/2019-02-04.md

```diff
@@ -0,0 +1,36 @@
+
+Feb. 4, 2019
+
+Secretary: Kevin Wheatley
+
+Attendees: Kevin Wheatley, John Mertic, Carl Rand, Larry Gritz, Michael “SPI” Dolan, Sean Cooper, Bernard Lefebvre, Doug Walker, Patrick Hodoul, Thanh Ha
+
```

Comment:
[Minor] I think that the list of attendees should be in the alphabetical order (perhaps also adding the Corporation name).